Fabulous.XamarinForms/src/Fabulous.XamarinForms/Xamarin.Forms.Core.json

```diff
@@ -2822,6 +2791,137 @@
       "primaryConstructorMembers": [
         "Items"
       ]
+    },
+    {
+      "type": "Xamarin.Forms.IndicatorView",
+      "name": "Indicator",
+      "properties": [
+        {
+          "source": "Count"
+        },
+        {
+          "source": "HideSingle"
+        },
+        {
+          "source": "IndicatorLayout"
+        },
+        {
+          "source": "IndicatorSize"
+        },
+        {
+          "source": "IndicatorTemplate"
+        },
+        {
+          "source": "IndicatorShape"
+        },
+        {
+          "source": "ItemsSource"
+        },
+        {
+          "source": "MaximumVisible"
+        },
+        {
+          "source": "Position"
+        },
+        {
+          "source": "SelectedIndicatorColor"
+        }
+      ]
+    },
+    {
+      "type": "Xamarin.Forms.SwipeItem",
+      "name": "SwipeItem",
+      "properties": [
+        {
+          "source": "BackgroundColor"
+        }
+      ],
+      "events": [
+        {
+          "source": "Invoked"
+        }
+      ]
+    },
+    {
+      "type": "Xamarin.Forms.SwipeItemView",
+      "name": "SwipeItemView",
+      "properties": [
+        {
+          "source": "Command"
+        }
+      ],
+      "events": [
+        {
+          "source": "Invoked"
+        }
+      ]
+    },
+    {
+      "type": "Xamarin.Forms.SwipeItems",
+      "name": "SwipeItems",
+      "properties": [
+        {
+          "source": "Item"
+        },
+        {
+          "source": "Mode"
+        },
+        {
+          "source": "SwipeBehaviorOnInvoked"
+        }
+      ],
+      "events": [
+        {
+          "source": "CollectionChanged",
+          "inputType": "obj -> unit",
+          "modelType": "System.Collections.Specialized.NotifyCollectionChangedEventHandler",
+          "convertInputToModel": "ViewConverters.makeSwipeItemsChangedEventHandler"
+        }
+      ]
+    },
+    {
+      "type": "Xamarin.Forms.SwipeView",
+      "name": "SwipeView",
+      "properties": [
+        {
+          "source": "BottomItems"
+        },
+        {
+          "source": "LeftItems"
+        },
+        {
+          "source": "RightItems"
+        },
+        {
+          "source": "TopItems"
+        }
+      ],
+      "events": [
```

Comment:
For all those 4 events, I think you can just set `"source": "EventName"`. The generator should be able to figure out the rest. (thanks to `System.EventHandler`/`System.EventHandler<T>`)
(So no need for ViewConverters too)